--- bamf-0.2.104/lib/libbamf/bamf-application.c.build	2017-09-21 21:41:32.528233899 +0200
+++ bamf-0.2.104/lib/libbamf/bamf-application.c	2017-09-21 21:41:38.144354981 +0200
@@ -179,7 +179,7 @@ bamf_application_get_windows (BamfApplic
     {
       view = l->data;
     
-      if (BAMF_IS_WINDOW (view));
+      if (BAMF_IS_WINDOW (view))
         {
           windows = g_list_prepend (windows, view);
         }
